docs/sources/azureeventhub.md

```diff
@@ -2,6 +2,10 @@
 
 This event source is meant to be used as a Container Source with a Knative cluster to consume messages from a Azure Event Hub and send them to a Knative service/function.
 
+## Prerequisite(s)
+
+- Azure Event Hubs & Azure Event Grid
+
```

Comment:
I also think this doc is not linked, and if it were the contents do not expose end user documentation.
Would you mind removing it as part of your PR?

Reply:
@odacremolbap are you meaning to delete the whole file?

Reply:
@odacremolbap I can delete the files if that's what you would like?

Reply:
yes delete